packages/application/style/icons.css

```diff
@@ -168,11 +168,6 @@
 }
 
 
-.jp-NotebookRunningIcon {
```

Comment:
There is no icon now, should this be re-instated with `--jp-icon-book`?

Reply:
Isn't that taken care of the rule just above?